### PR TITLE
Portfolio   mobile menu

### PR DESCRIPTION
**Pull request by LethaboMot**
in the project we:

Added HTML to add the contact portfolio and about me
Added CSS for the style of the overlay and the transition
Added javascript for the button to react to the information of the CSS and html

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,7 +99,7 @@ After you're finished please remove all the comments and instructions!
 
 > Describe between 1-3 key features of the application.
 
-- **[desktop version]**
+- **[making an overlay]**
 - **[key_feature_2]**
 - **[key_feature_3]**
```
Comment (Munsa1) [inline README.md:104]: - [ ] You could make your `key features` more descriptive, for example you could say:
- Added a mobile menu
- Added links to specific sections of the site on the mobile menu
- Added an overlay to the mobile menu
- [ ] This is more descriptive thaan what we currently have, kindly fix this

File: README.md [hunk 2]
```diff
@@ -205,12 +205,17 @@ Example:
...
 - Twitter: [@profcipher](https://twitter.com/profcipher)
 - LinkedIn: [Lethabo Motloung](https://linkedin.com/in/linkedinhandle)
 
+👤 **Enis Memic**
+
+- GitHub: [@enis-memic](https://github.com/enis-memic)
+- LinkedIn: [Enis Memic](https://linkedin.com/in/enis-memic)
+
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (Munsa1) [inline README.md:220]: - [ ] Your `future features` section is missing, kindly add this section and make it as descriptive as possible. You could add the futture features you will implement in the milestones to come.

File: script.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+function openNav(open) {
+    document.getElementById('slidemenu').style.width = '100%';
+    return open;
+  }
+  openNav('hello');
```
Comment (Munsa1) [inline script.js:5]: - [ ] We have an error in the console pointing to line `2` and line `3` that is causing the page not refresh, kindly fix this error
image

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @LethaboMot, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Passing Linters :heavy_check_mark: 
Correct GitHub Flow :heavy_check_mark: 
Descriptive ReadMe File :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._
### Due to insufficient files I will leave a comment here
- [ ] Kindly add linter configurations for JavaScript files as this is a requirement.
image
- As you can see ` eslint` checks are missing, kindly fix this
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Gambit142 — APPROVED** [review]
Hi @LethaboMot @enis-memic 👋,

## Status: Approved 🟢   

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

### Highlights
- All linters checks passed ✔️ 
- Mobile menu functions properly ✔️ 

### Optional Suggestion
- Kindly consider the suggestion tagged _**[OPTIONAL]**_ under the review. 👍 💻 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by topeogunleye — APPROVED** [review]
### Hi @LethaboMot and @enis-memic,

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good description :heavy_check_mark:
 
## [Optional] suggestions:
* Nothing to mention.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**